archetypes/Portfolio/OverviewTable/styles.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+import { HiddenExpand as UnstyledHiddenExpand } from '~/components/General';
+import Button from '~/components/General/Button';
+import { SearchInput as UnstyledSearchInput } from '~/components/General/SearchInput';
+import { TableRowCell } from '~/components/General/TWTable';
+import ArrowRight_ from '~/public/img/general/arrow-right.svg';
+import { Theme } from '~/store/ThemeSlice/themes';
+
+export const Container = styled.div`
+    border-radius: 0.75rem;
+    box-shadow: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
+    background: ${({ theme }) => theme.background.primary};
+`;
+
+export const Wrapper = styled.div`
+    white-space: nowrap;
+    padding: 1rem;
+
+    @media ${({ theme }) => theme.device.lg} {
+        display: flex;
+        justify-content: space-between;
+    }
+`;
+
+export const TitleContent = styled.div`
+    display: flex;
+    gap: 0.75rem;
+`;
+
+export const Title = styled.div`
+    line-height: 150%;
+    font-weight: 600;
+    font-size: 20px;
```

Comment:
Update font-sizes to theme.fontSize throughout this file

Reply:
done